src/django_upgrade/ast.py

```diff
@@ -74,7 +74,7 @@ def is_passing_comparison(
         and isinstance(test.ops[0], (ast.Gt, ast.GtE, ast.Lt, ast.LtE))
         and len(test.comparators) == 1
         and isinstance((comparator := test.comparators[0]), ast.Tuple)
-        and len(comparator.elts) == 2
+        and 0 < len(comparator.elts) <= 2
```

Comment:
A little more obvious
```suggestion
        and 1 <= len(comparator.elts) <= 2
```